components/ItemGrid.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div>
+    <v-row v-if="loading">
+      <v-col cols="12" class="card-grid-container">
+        <skeleton-card v-for="n in 24" :key="n" />
+      </v-col>
+    </v-row>
+    <dynamic-scroller
+      v-if="items.length"
+      class="scroller"
+      :items="itemsChunks"
+      :min-item-size="350"
+      :buffer="$vuetify.breakpoint.height * 1.5"
+      page-mode
+    >
+      <template #default="{ item, index, active }">
+        <dynamic-scroller-item
+          :item="item"
+          :active="active"
+          :data-index="index"
+          class="card-grid-container"
+        >
+          <card v-for="card of item.chunk" :key="card.Id" :item="card" />
+        </dynamic-scroller-item>
+      </template>
+    </dynamic-scroller>
+    <v-row v-else-if="!loading" justify="center">
+      <v-col cols="12" class="card-grid-container empty-card-container">
+        <skeleton-card v-for="n in 24" :key="n" boilerplate />
+      </v-col>
+      <div class="empty-message text-center">
+        <slot>
+          <h1 class="text-h5">
+            {{ $t('noResultsFound') }}
+          </h1>
+        </slot>
+      </div>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { chunk } from 'lodash';
+import Vue from 'vue';
+import { BaseItemDto } from '~/api';
+
+export default Vue.extend({
+  props: {
+    items: {
+      type: Array,
+      required: true
+    },
+    loading: {
+      type: Boolean,
+      required: true
+    }
+  },
+  computed: {
+    itemsChunks(): Array<{ [id: number]: BaseItemDto }> {
+      let cardsPerLine = 8;
+
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Why are we still having this after my fix? I've got no linter errors back when I committed and everything worked flawlessly.

I can no longer go back for it because you force pushed.

Reply:
Indeed, I forgot to re-remove them when porting over your change (The commit you had pushed undid some changes I had made before, so I re-applied the changes on top of the cleaned-up branch).

I just pushed a commit that removes these `ts-ignore`.